specs/ReleaseChannelPreferenceSet.md

```diff
@@ -0,0 +1,325 @@
+        runtimeChannelPreferenceSet: channels
+    );
+    // Use GetAvailableCoreWebView2BrowserVersionStringWithOptions to check which
+    // channel is used with the custom options. If Beta channel was not found,
+    // install it on the device.
+    string version =
+        CoreWebView2Environment.GetAvailableBrowserVersionStringWithOptions(
+            browserExecutableFolder: null,
+            environmentOptions: customOptions);
+    if (!IsBetaChannel(version))
+    {
+        InstallChannel(CoreWebView2RuntimeChannel.Beta);
+    }
+    // If the loader is unable to find a valid installation from the runtime
+    // channel preference set, environment creation will fail.
+    _webViewEnvironment = await CoreWebView2Environment.CreateAsync(
+        options: customOptions);
+    if (_webViewEnvironment != null)
+    {
+        CheckChannel(_webviewEnvironment.BrowserVersionString());
+    }
+}
+```
+```cpp
+void AppWindow::InitializeWebViewEnvironment()
+{
+    auto options = Microsoft::WRL::Make<CoreWebView2EnvironmentOptions>();
+    CHECK_FAILURE(options->put_ShouldReverseChannelSearchOrder(TRUE));
+    COREWEBVIEW2_RUNTIME_CHANNEL channels =
+        COREWEBVIEW2_RUNTIME_CHANNEL_BETA |
+        COREWEBVIEW2_RUNTIME_CHANNEL_STABLE;
+    CHECK_FAILURE(options->SetRuntimeChannelPreferenceSet(channels));
+    // Use GetAvailableCoreWebView2BrowserVersionStringWithOptions to check which
+    // channel is used with the custom options. If Beta channel was not found,
+    // install it on the device.
+    wil::unique_cotaskmem_string version;
+    HRESULT hr = GetAvailableCoreWebView2BrowserVersionStringWithOptions(
+        nullptr, options.Get(), &version);
+    if (!IsBetaChannel(version.get()))
+    {
+        InstallChannel(COREWEBVIEW2_RUNTIME_CHANNEL_BETA);
+    }
+    // If the loader is unable to find a valid installation from the runtime
+    // channel preference set, environment creation will fail with
+    // HRESULT_FROM_WIN32(ERROR_FILE_NOT_FOUND).
+    HRESULT hr = CreateCoreWebView2EnvironmentWithOptions(
+        nullptr, m_userDataFolder.c_str(), options.Get(),
+        Callback<ICoreWebView2CreateCoreWebView2EnvironmentCompletedHandler>(
+            this, &AppWindow::OnCreateEnvironmentCompleted)
+            .Get());
+}
+void AppWindow::OnCreateEnvironmentCompleted(
+    HRESULT result, ICoreWebView2Environment* environment)
+{
+    if (environment)
+    {
+        wil::unique_cotaskmem_string version;
+        CHECK_FAILURE(environment->get_BrowserVersionString(&version));
+        CheckChannel(version.get());
+    }
+}
+```
+# API Details
+```
+/// The WebView2 Runtime channel. Use `RuntimeChannelPreferenceSet` and
+/// `ShouldReverseChannelSearchOrder` on `ICoreWebView2EnvironmentOptions` to
+/// control which channel the WebView2 loader searches for.
+/// |Channel|Primary purpose|How often updated with new features|
+/// |:---:|---|:---:|
+/// |Stable (WebView2 Runtime)|Broad Deployment|~4 weeks|
```

Comment:
Is the release cadence of the different channels something that needs to be encoded in the docs for this API?  Hopefully there is already a docs.microsoft.com page that documents the different channels and their update cadence.